File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/First.java
package jp.ac.dendai.c.jtp.adventuresample.scene;
import jp.ac.dendai.c.jtp.adventuresample.R;
public class First extends AbstractScene {
    @Override
    public int getImageId() {
        return R.drawable.first;
    }
    @Override
    public int getMessageId() {
        return R.array.message1;
    }
    @Override
    public int getQuestionId() {
        return 0;
    }
    @Override
    public GameState next(int no) {
        return GameState.second;
    }
    @Override
    public int getDateId() {
        return R.string.date1;
    }
    @Override
    public int getMusicId() { return R.raw.owarininokorumono; }
}

File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/First.kt
package jp.ac.dendai.c.jtp.adventuresample.scene

import jp.ac.dendai.c.jtp.adventuresample.R

class First : AbstractScene() {
    override val imageId = R.drawable.first

    override val messageId = R.array.message1

    override val questionId = 0

    override fun next(no: Int): GameState = GameState.second

    override val dateId = R.string.date1

    override val musicId = R.raw.owarininokorumono
}